crates/cli-support/src/wit/mod.rs

```diff
@@ -476,22 +486,7 @@ impl<'a> Context<'a> {
             return Ok(());
         }
 
-        let prev_start = match self.module.start {
-            Some(f) => f,
-            None => {
-                self.module.start = Some(id);
-                return Ok(());
-            }
-        };
-
-        // Note that we call the previous start function, if any, first. This is
-        // because the start function currently only shows up when it's injected
-        // through thread/externref transforms. These injected start functions
-        // need to happen before user code, so we always schedule them first.
-        let mut builder = walrus::FunctionBuilder::new(&mut self.module.types, &[], &[]);
-        builder.func_body().call(prev_start).call(id);
-        let new_start = builder.finish(Vec::new(), &mut self.module.funcs);
-        self.module.start = Some(new_start);
+        self.module.exports.add("__wbindgen_main", id);
```

Comment:
I didn't actually test if the normal `init` function works now, for some reason `__wbindgen_main` doesn't end up in the final WASM file. Is it being cleaned up somehow? Or is `module.exports.add()` not sufficient?

Reply:
Indeed it was being cleaned up in `unexport_intrinsics`, I hope the way I fixed this is appropriate.